dist/PublicLab.Editor.css

```diff
@@ -42,6 +42,24 @@
   text-align: center;
 }
 
+.btn-info {
+  background: transparent !important;
+  padding: 1px !important;
+  color: #337AB7;
```

Comment:
Is the usage of `!important` really necessary here? `!important` should be avoided.

Reply:
btn-info is overriding bootstrap class. We don't have latest version of bootstrap 4.4.1. In this version we can use border-0 and p-0 for changing margins but not in the version 3.4.1. !important is necessary

Reply:
Maybe you could use `btn-light`. It's very similar to your custom CSS.

Reply:
@niravasher the rules `btn-light` is unnecessary. Please remove it.